source/GameInterface/Services/MobileParties/Patches/Disable/DisableRecruitmentCampaignBehavior.cs

```diff
@@ -1,12 +0,0 @@
-﻿using HarmonyLib;
-using TaleWorlds.CampaignSystem.CampaignBehaviors;
-
-namespace GameInterface.Services.MobileParties.Patches;
-
-
-[HarmonyPatch(typeof(RecruitmentCampaignBehavior))]
-internal class DisableRecruitmentCampaignBehavior
-{
-    [HarmonyPatch(nameof(RecruitmentCampaignBehavior.RegisterEvents))]
```

Comment:
Keep mercinary recruitment disabled in this class or implement along with this issue.

Also disable ticking for for clients

Reply:
Mercenary recruitment looks to be included in this. Not tested yet.

Reply:
This file does not exist for me..